### DISCUSSION TITLE
Import from Obsidian markdown

### DISCUSSION THREAD

**Discussion by jaroet** [TriliumNext/Trilium#3383] [General]
I am trying Trilium Notes (again after a few years) coming from Obsidian. When I import the Obsidian markdown files they are imported in Trilium and the text is rather looking good. Using the search function with the predefined actions like 'add label' or 'move items' I can add attributes to certain notes and move them depending on those attributes to certain folders. 

The big missing option here is to rename the [[]] syntax from obsidian to the @ syntax in Trilium. My links to pages in Obsidian are between [[]] and I would like to make them Trilium notes you normally add using the @ or the META-J shortcut. 

I have about 1180 pages so manually changing those would be a very large commitment and quit time-consuming. I see there is an option for 'Execute script' in the actions menu from the search. 

I tried searching for this but I see that most conversion are the other way around from Trilium to Obsidian where the Markdown exporter does a good enough job I think (have not tested it). 

Has anybody ever went from Obsidian to Trilium and if so how did they convert there existing Obsidian notes to Trilium. Any information about this would be appreciated very much.

**Comment by zadam** [comment]
I would recommend trying to find a way to export Obsidian MD to HTML which should be then importable into Trilium.

**Reply by jaroet** [reply]
Thanks for the suggestion. I tried with different options also importing as HTML. I can get the text and the external links pretty clean but the big issue is the internal links (to other notes/pages). Obsidian uses [[internal link]] syntax. I would need a script that links that what is between the [[ and the ]] to a note with the name equal to the text between those. I am not sure how to create a link in a script, if possbile at all.
